.storybook/preview.tsx

```diff
@@ -0,0 +1,50 @@
+import { IntlProvider } from 'react-intl';
+
+import features from '../examples/src/features';
+
+import '../src/styles/variables';
+import '../src/styles/base.scss';
+import {reactIntl} from "./reactIntl";
+import {ProjectAnnotations, Renderer} from "@storybook/types";
```

Comment:
spaces between curly brackets for these imports. does this file maybe need a quick format?